sycl/include/sycl/ext/intel/math.hpp

```diff
@@ -49,9 +49,7 @@ _iml_half_internal __imf_truncf16(_iml_half_internal);
 
 namespace sycl {
 __SYCL_INLINE_VER_NAMESPACE(_V1) {
-namespace ext {
-namespace intel {
-namespace math {
+namespace ext::intel::math {
 
 #if __cplusplus >= 201703L
```

Comment:
Do we still intended to support compatibility with C++14 mode? I see that some C++17 features are being used unconditionally.

Reply:
No, we require at least C++17 since https://github.com/intel/llvm/pull/6678. I'm removing the guards in https://github.com/intel/llvm/pull/7687.